packages/analyzer/src/utils/manifest-helpers.js

```diff
@@ -140,3 +140,25 @@ export function getModuleForClassLike(cem, className) {
   return result;
 }
 
+/**
+ * Given a manifest module, a class name, and a class member name, gets the 
+ * manifest doc for the module's class' member.
+ *
+ * @param  {Partial<import('custom-elements-manifest/schema').Module>} moduleDoc Manifest module
+ * @param  {string} className Class to get member of
+ * @param  {string} memberName Class member to get
+ * @return {import('custom-elements-manifest/schema').ClassMember|void} the requested class member
+ */
+export function getClassMemberDoc(moduleDoc, className, memberName) {
+  /** @type {import('custom-elements-manifest/schema').ClassDeclaration} */
+  const classDoc = (moduleDoc.declarations.find(x => x.name === className));
```

Comment:
Note that this wont find classes that are exported as default

Reply:
should deal with that in https://github.com/webcomponents/custom-elements-manifest/issues/72, right?

Reply:
also, with current spec, if user passes in `default` for `className`, then we're good, right?

Reply:
Yeah thats true, just something to keep in mind 🙂